autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml

```diff
@@ -68,23 +68,23 @@
           safety_buffer_lateral: 0.7
           safety_buffer_longitudinal: 0.0
         bicycle:
-          enable: false
+          enable: true
           moving_speed_threshold: 0.28                 # 1.0km/h
           moving_time_threshold: 1.0
           max_expand_ratio: 0.0
           envelope_buffer_margin: 0.8
           safety_buffer_lateral: 1.0
           safety_buffer_longitudinal: 1.0
         motorcycle:
-          enable: false
+          enable: true
```

Comment:
couldn't find description fot this param in `avoidance.param.yaml`
this means like `detection_enable`?

Reply:
This param means "enable avoidance maneuver for motorcycle".

> couldn't find description fot this param in avoidance.param.yaml
Sorry. I'll update documentation in another PR as soon as possible.